mypyc/transform/integer_optimization.py

```diff
@@ -0,0 +1,19 @@
+from typing import List, Dict
+
+from mypyc.ir.ops import BasicBlock, LoadInt, Assign
+from mypyc.ir.rtypes import is_short_int_rprimitive, is_int_rprimitive
+
+
+def find_constant_integer_registers(blocks: List[BasicBlock],
+                                    convert_tagged: bool = False) -> Dict[str, int]:
+    """
```

Comment:
Style nit: move the first sentence to the same line as `"""` (e.g. `"""Find all ...`).